src/Aspire.Hosting/Postgres/PostgresBuilderExtensions.cs

```diff
@@ -62,6 +62,19 @@ public static IResourceBuilder<PostgresDatabaseResource> AddDatabase(this IResou
                                          .WithManifestPublishingCallback(context => WritePostgresDatabaseToManifest(context, postgresDatabase));
     }
 
+    /// <summary>
+    /// Adds a data volume mount to a PostgreSQL container resource (mounted to /var/lib/postgresql/data in the container).
+    /// </summary>
+    /// <typeparam name="T">The resource type.</typeparam>
+    /// <param name="builder">The resource builder.</param>
+    /// <param name="source">The source path of the volume. This is the physical location on the host.</param>
+    /// <param name="type">The type of volume mount.</param>
+    /// <returns>The <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<T> WithVolumeMountForData<T>(this IResourceBuilder<T> builder, string source, VolumeMountType type = default) where T : PostgresContainerResource
```

Comment:
Should the isReadonly-Parameter from WithVolumeMount also be available for this method? Are there any common cases where this could be usefull?